docs/Concepts/Plugins/Plugins.md

```diff
@@ -46,13 +46,18 @@ geth <other parameters> \
 ## Plugin Integrity Verification
 
 Plugin Central Server can be used to download and verify plugin integrity using [PGP](https://en.wikipedia.org/wiki/Pretty_Good_Privacy).
-The architecture enables the same verification process locally via `--plugins.localverify` and `--plugins.publickey` flags or
-remotely with custom plugin central - reference the [`Settings`](../../HowTo/Configure/Plugins.md) section for more information on how to support custom plugin central.
+The architecture enables the same verification process locally via
+[`--plugins.localverify`](../../Reference/CLI-Syntax.md#pluginslocalverify) and
+[`--plugins.publickey`](../../Reference/CLI-Syntax.md#pluginspublickey) flags or
+remotely with custom plugin central - reference the [`Settings`](../../HowTo/Configure/Plugins.md) section for more
+information on how to support custom plugin central.
 
-If the flag `--plugins.skipverify` is provided at runtime the plugin verification process will be disabled.
+If the flag [`--plugins.skipverify`](../../Reference/CLI-Syntax.md#pluginsskipverify) is provided at runtime the plugin
+verification process is disabled.
 
 !!! warning
-    Using `--plugins.skipverify` is not advised for production settings and it should be avoided as it introduces security risks.
+
+    We do not recommend using `--plugins.skipverify` for production settings as it introduces security risks.
```

Comment:
We use the Microsoft style manual which suggests using common contractions. So this could be, "We don't..."

Alternatively rephrase to, "Using `--plugins.skipverify` introduces security risks and isn't recommended for production environments."